src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/SubaccountSupervisor.kt

```diff
@@ -258,40 +259,81 @@ internal class SubaccountSupervisor(
         val url = helper.configs.privateApiUrl(if (configs.useParentSubaccount) "parent-historical-pnl" else "historical-pnl") ?: return
         val params = if (configs.useParentSubaccount) parentSubaccountParams() else subaccountParams()
 
-        val historicalPnl = helper.parser.asNativeList(
-            helper.parser.value(
-                stateMachine.data,
-                "wallet.account.subaccounts.$subaccountNumber.historicalPnl",
-            ),
-        )?.mutable()
-
-        if (historicalPnl != null) {
-            val last = helper.parser.asMap(historicalPnl.lastOrNull())
-            if (helper.parser.asBool(last?.get("calculated")) == true) {
-                historicalPnl.removeLast()
+        if (stateMachine.staticTyping) {
+            val historicalPNLs = stateMachine.internalState.wallet.account.subaccounts[subaccountNumber]?.historicalPNLs
+            // TODO: remove last if calculated
```

Comment:
We don't calculate the last tick from the client for v4 as on v3, so this is just a placeholder for now